ansible_collections/arista/avd/roles/eos_designs/defaults/main/main.yml

```diff
@@ -111,6 +111,11 @@ platform_settings:
     reload_delay:
       mlag: 900
       non_mlag: 1020
+  - platforms: [ '7368X4']
```

Comment:
Space needed at the end for consistency. **[ '7368X4' ]**

Reply:
interesting - it passed the pre-commit check like this. Fixed in next commit